flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamExecDataStreamScan.scala

```diff
@@ -110,7 +110,6 @@ class StreamExecDataStreamScan(
     val config = planner.getTableConfig
```

Comment:
Remove `getSourceTransformation()` method in this file. It is never used.

Reply:
ok